eng/pipelines/coreclr/runtime-nativeaot-outerloop.yml

```diff
@@ -0,0 +1,196 @@
+
+# This pipeline includes the Native AOT tests that we don't have resources to
+# run on every PR but that we want to still have test coverage, we run this
+# pipeline on a schedule and also developers can run it via /azp run command on
```

Comment:
Just a clarification, to initiate a build manually, one would need to do: `/azp run runtime-nativeaot-outerloop` ?

Reply:
Yup that's the intent